### ISSUE TITLE
Failing "staticland domain" call still has 0 exit code

### ISSUE THREAD

**Issue by ambirdsall** [staticland/staticland#11]
I haven't done any other auditing of exit codes; just happened to notice this.

Output from my terminal earlier today:
``` sh
>  staticland -v
3.2.1
>  staticland domain ambirdsall.space; echo $?

  FAILED!
    Domain not ready to deploy to api.static.land. Currently at IP 162.255.119.250.

0
>  if staticland domain ambirdsall.space; then echo 'Success exit code'; fi

  FAILED!
    Domain not ready to deploy to api.static.land. Currently at IP 162.255.119.250.

Success exit code
```

A nonzero exit in this scenario would be useful for scripting deployment.

By way of `man sysexits`, I found a list of conventional exit codes at [`/usr/include/sysexits.h`](https://linux.die.net/include/sysexits.h). This one seems right for the job:

``` c
/*
 *	EX_TEMPFAIL -- temporary failure, indicating something that
 *		is not really an error.  In sendmail, this means
 *		that a mailer (e.g.) could not create a connection,
 *		and the request should be reattempted later.
 */

#define EX_TEMPFAIL	75	/* temp failure; user is invited to retry */
```

**Comment by sethvincent** [comment]
:+1: I'll make sure this is included in the next version.

**Comment by sethvincent** [comment]
This is fixed in v3.2.2. Thanks!
